extensions/wikia/ThemeDesigner/templates/ThemeDesigner_Picker.php

```diff
@@ -12,78 +12,12 @@ class="color-name">
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/opulence_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/opulence.png">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/oasis_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/oasis.png">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/babygirl_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/babygirl.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/carbon_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/carbon.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/rockgarden_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/rockgarden.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/creamsicle_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/creamsicle.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/plated_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/plated.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/police_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/police.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/aliencrate_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/aliencrate.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/sapphire_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/sapphire.png">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/sky_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/sky.png">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/moonlight_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/moonlight.jpg">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/beach_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/beach.png">
-			</li>
-			<li>
-				<img src="<?= $wg->ExtensionsPath ?>/wikia/ThemeDesigner/images/dragstrip_swatch.jpg"
-					 data-image="<?= $wg->ResourceBasePath ?>/skins/oasis/images/themes/dragstrip.jpg">
-			</li>
+			<?php foreach (ThemeDesignerBackgroundAssets::$backgrounds as $bg) { ?>
+				<li>
+					<img src="<?= $wg->ExtensionsPath . $bg['thumbnail'] ?>"
+					     data-image="<?= $wg->ResourceBasePath . $bg['source'] ?>">
+				</li>
+			<?php } ?>
```

Comment:
I think using the alternative syntax (`foreach(...):`) above and `endforeach;` here is slightly more readable